### PR TITLE
Line Text Format Cleanup

### PR DESCRIPTION
**Pull request by steward379**
- if there's no tags, there won't be unnecessary empty ``;

### CODE REVIEW

File: lib/notification/line.ts [hunk 1]
```diff
@@ -11,6 +11,10 @@ const generateHouseMessage = (house: DataItem) => {
 
   const coverPhoto = getCoverPhoto(house);
 
+  let rentTag;
+
+  house.rent_tag ? (rentTag = `\`${house.rent_tag?.map((tag) => tag.name)}\``) : (rentTag = ``);
```
Comment (Yukaii) [inline lib/notification/line.ts]:
```suggestion
  const rentTag =  house.rent_tag ? `\`${house.rent_tag?.map((tag) => tag.name)}\``) : '';
```

**Review by Yukaii — CHANGES_REQUESTED** [review]

**Review by moojing — APPROVED** [review]

**Comment by moojing** [comment]
branch murderer 🤯

**Review by Yukaii — APPROVED** [review]
<ruby>LGTM<rt>Looks Good To Me</rt></ruby>

**Pull request merged**